plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/PredicatePushdownController.java

```diff
@@ -47,10 +47,9 @@
             return FULL_PUSHDOWN.apply(session, domain);
         }
 
-        if (domain.getValues().isDiscreteSet()) {
```

Comment:
I think we can retain this `if`. If `! domain.getValues().isDiscreteSet()` then it surely `! simplifiedDomain.getValues().isDiscreteSet()`.